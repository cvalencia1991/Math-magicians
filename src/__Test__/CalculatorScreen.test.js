import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Calculatorscren from '../Pages/Calulatorscreen';

describe('Testing Calculator page', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Calculatorscren />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
