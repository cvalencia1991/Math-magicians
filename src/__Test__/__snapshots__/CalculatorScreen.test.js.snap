// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Calculator page renders correctly 1`] = `
<div>
  <h2>
    Let's do some math!
  </h2>
  <div
    className="positionCal"
  >
    <span
      className="styleoutput"
    >
      0
    </span>
    <div
      className="stylebuttoms"
    >
      <div
        className="styleCalculator"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          AC
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          7
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          4
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          1
        </button>
      </div>
      <div
        className="styleCalculator"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          +/-
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          8
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          5
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          2
        </button>
      </div>
      <div
        className="styleCalculator"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          %
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          9
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          6
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          3
        </button>
      </div>
      <div
        className="styleCalculator"
      >
        <button
          className="buttonfunction"
          onClick={[Function]}
          type="button"
        >
          รท
        </button>
        <button
          className="buttonfunction"
          onClick={[Function]}
          type="button"
        >
          *
        </button>
        <button
          className="buttonfunction"
          onClick={[Function]}
          type="button"
        >
          -
        </button>
        <button
          className="buttonfunction"
          onClick={[Function]}
          type="button"
        >
          +
        </button>
      </div>
    </div>
    <div
      className="stylebottomcalculator"
    >
      <button
        className="buttonZero"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="buttonfunction"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
